server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -179,10 +177,10 @@ public void testExpandsConfigurationWhenGrowingFromThreeToFiveNodesAndShrinksOnF
 
         final ClusterNode leader = cluster.getAnyLeader();
 
-        logger.info("setting fault tolerance to 1");
-        leader.submitSetMasterNodesFailureTolerance(1);
+        logger.info("setting auto-shrink reconfiguration to true");
+        leader.submitSetAutoShrinkVotingConfiguration(true);
         cluster.stabilise(DEFAULT_CLUSTER_STATE_UPDATE_DELAY);
-        assertThat(CLUSTER_MASTER_NODES_FAILURE_TOLERANCE.get(leader.getLastAppliedClusterState().metaData().settings()), equalTo(1));
+        assertTrue(CLUSTER_AUTO_SHRINK_VOTING_CONFIGURATION.get(leader.getLastAppliedClusterState().metaData().settings()));
```

Comment:
maybe also update the test method name to say that it only shrinks back to 3?

Reply:
Ok I added `BackToThree`.